chat/chat.py

```diff
@@ -94,6 +94,12 @@ class ChatXBlock(StudioEditableXBlockMixin, XBlock):
         scope=Scope.content,
     )
 
+    enable_restart_button = Boolean(
+        display_name=_("Enable restart button"),
+        default=False,
```

Comment:
@replaceafill It seems unusual to me too, but that's what the Studio block property editor  uses for boolean fields for some reason. If you check some other block types that contain boolean settings (even problem types included in edx-platform), you will see that they consistently appear as True/False dropdowns in the Studio.